packages/interpretations/src/api/redirect.js

```diff
@@ -58,42 +57,30 @@ export const itemTypeMap = {
         id: REPORT_TABLE,
         appUrl: (modelId, interpretationId) => `dhis-web-pivot/?id=${modelId}&interpretationid=${interpretationId}`,
         propName: 'reportTable',
-        appName: i18n.t('Pivot Tables'),
```

Comment:
These props were removed and instead placed in the only locations where they were used.